sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -38,20 +38,10 @@ For examples of Logs and Metrics queries, see the [Examples](#examples) section.
 
 ## Key concepts
 
-### Logs query rate limits and throttling
```

Comment:
Keep the "Logs query rate limits and throttling" and "Metrics data structure" sections here. They're present in the other languages' READMEs. For example: https://github.com/Azure/azure-sdk-for-java/blob/6a429425c0a498469a368270d77fa02f673adc9f/sdk/monitor/azure-monitor-query/README.md#L93-L106